src/modules/launcher/Plugins/Microsoft.Plugin.Program.UnitTests/Programs/Win32Tests.cs

```diff
@@ -287,7 +287,7 @@ public void DedupFunctionWhenCalledMustRemoveInternetShortcuts()
         }
 
         [TestMethod]
-        public void DedupFunctionWhenCalledMustNotRemovelnkWhichdoesNotHaveExe()
+        public void DedupFunctionWhenCalledMustNotRemovelnkWhichDoesNotHaveExe()
```

Comment:
I'm pretty sure this is an auto thing, and there didn't seem to be any reason not to do:
```diff
-Whichdoes
+WhichDoes
```